### ISSUE TITLE
Warn on use of numeric comparison without explicit `test`or '['

### ISSUE THREAD

**Issue by aderuelle** [koalaman/shellcheck#1514]
#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/bin/ksh
if ${VAR} -eq "yes"
then
        echo "ok"
fi

```

#### Here's what shellcheck currently says:

no output

#### Here's what I wanted or expected to see:
```
In - line 2:
if ${VAR} -eq "yes"
               ^-- SCxxxx: Suspicious use of -eq switch on ${VAR} whithout explicit test command or bracket '['
```
